client/state/data-layer/wpcom-http/actions.js

```diff
@@ -17,10 +35,11 @@ import { WPCOM_HTTP_REQUEST } from 'state/action-types';
  * @param {Object} [onProgress] Redux action to call on progress events from an upload
  * @param {Object} [options] extra options to send to the middleware, e.g. retry policy or offline policy
  * @param {Object} [action] default action to call on HTTP events
- * @returns {Object} Redux action describing WordPress.com API HTTP request
+ * @return {Object} Redux action describing WordPress.com API HTTP request
```

Comment:
I used to use `@return` instead of `@returns`. In fact, the number of `@return` is clearly greater than `@returns` in the base code of calypso. And it seems that both are accepted.